### PR TITLE
Feature/border

### PR DESCRIPTION
**Pull request by adricampo**
Add a border to the tittle

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,9 +1,12 @@
 body {
    background-color: aquamarine;
-   
+   border: purple 5px solid;
```
Comment (MartaPValls) [inline style.css]: is not purple, should be pink

**Comment by adricampo** [comment]
modified !

File: style.css [hunk 2]
```diff
@@ -1,9 +1,12 @@
 body {
```
Comment (MartaPValls) [inline style.css]: Ok

Comment (adricampo) [inline style.css]: 👍🏻

**Pull request merged**